README.md

```diff
@@ -287,6 +287,19 @@ var result = [
 ];
 ```
 
+#### Autoprefixer: vendor prefixes
+
+By default the preprocessor uses the following [browserslist](https://github.com/ai/browserslist) configuration:
+
+```
+> 1%, last 2 versions, safari > 6, ie > 9, ios > 6, android > 4.3, samsung > 3, chromeandroid > 50
+```
```

Comment:
Perhaps mention here somewhere that this is to match the SUIT browser support table?